src/lando/api/tests/test_try.py

```diff
@@ -15,6 +15,9 @@
 from lando.api.legacy.workers.landing_worker import LandingWorker
 from lando.main.models.landing_job import LandingJob, LandingJobStatus
 
+
+pytest.skip(allow_module_level=True)
```

Comment:
Nit: should probably add a TODO here so this doesn't fall off the radar.

Reply:
There's a few of these right now, I'll file a bug to make sure we unskip all of them.